recipe/build.sh

```diff
@@ -5,8 +5,8 @@ LLVM_PREFIX=$PREFIX
 if [[ "$target_platform" == osx-* ]]; then
     export CFLAGS="$CFLAGS -isysroot $CONDA_BUILD_SYSROOT"
     export CXXFLAGS="$CXXFLAGS -isysroot $CONDA_BUILD_SYSROOT"
-    export LDFLAGS="$LDFLAGS -isysroot $CONDA_BUILD_SYSROOT"
-    export INSTALL_NAME_TOOL="/usr/bin/install_name_tool"
+    export LDFLAGS="$LDFLAGS -isysroot $CONDA_BUILD_SYSROOT -framework CoreFoundation"
+
     export CMAKE_EXTRA_ARGS="-DCMAKE_OSX_SYSROOT=$CONDA_BUILD_SYSROOT -DLIBCXX_ENABLE_VENDOR_AVAILABILITY_ANNOTATIONS=ON"
```

Comment:
```suggestion
    export CMAKE_EXTRA_ARGS="-DLIBCXX_ENABLE_VENDOR_AVAILABILITY_ANNOTATIONS=ON"
```
The OSX sysroot is already being set with clang and the sysroot itself. Shouldn't need it here, right?